recipe/patches/0001-Do-not-force-pickle5-in-sys.path.patch

```diff
@@ -1,31 +0,0 @@
-From 76df41d9f33447df2309183b0ad18f56c8fe788a Mon Sep 17 00:00:00 2001
-From: Vasily Litvinov <vasilij.n.litvinov@intel.com>
-Date: Thu, 5 Nov 2020 12:04:25 +0300
-Subject: [PATCH 1/6] Do not force pickle5 in sys.path
-
-Signed-off-by: Vasily Litvinov <vasilij.n.litvinov@intel.com>
----
- python/ray/__init__.py | 6 ------
- 1 file changed, 6 deletions(-)
-
-diff --git a/python/ray/__init__.py b/python/ray/__init__.py
-index 758356f1a..6702014c5 100644
---- a/python/ray/__init__.py
-+++ b/python/ray/__init__.py
-@@ -34,13 +34,6 @@ def _configure_system():
-                      "override this by explicitly setting OMP_NUM_THREADS.")
-         os.environ["OMP_NUM_THREADS"] = "1"
- 
--    # Add the directory containing pickle5 to the Python path so that we find
--    # the pickle5 version packaged with ray and not a pre-existing pickle5.
--    pickle5_path = os.path.join(
--        os.path.abspath(os.path.dirname(__file__)), "pickle5_files"
--    )
--    sys.path.insert(0, pickle5_path)
```

Comment:
is there a reason this patch is removed?

Reply:
Yes, we do not update the sys path anymore in setup.py, so the patch is not needed anymore